MarsRoverBestGroup3.0/MarsRoverBestGroup3.0/APICall.cs

```diff
@@ -15,7 +15,8 @@ public class APICall
         private RestClient _restClient;
         public APICall()
         {
-            _apiKey = Environment.GetEnvironmentVariable("API_KEY");
+            //_apiKey = Environment.GetEnvironmentVariable("API_KEY");
+            _apiKey = ConfigurationManager.AppSettings["APIKey"];
```

Comment:
You should set this back to the GetEnvironmentVariable version, and ask Joseph for the dotenv file so you can make a local one :)